.travis.yml

```diff
@@ -1,36 +1,42 @@
 language: python
+
 sudo: false
+
 python:
 - '2.7'
+- '3.5'
+- '3.6'
+- '3.7'
+
 install:
-- if [[ "$TRAVIS_PYTHON_VERSION" == "2.7" ]]; then wget http://repo.continuum.io/miniconda/Miniconda-3.7.0-Linux-x86_64.sh
-  -O miniconda.sh; else wget http://repo.continuum.io/miniconda/Miniconda3-3.7.0-Linux-x86_64.sh
-  -O miniconda.sh; fi
+- wget http://repo.continuum.io/miniconda/Miniconda3-3.7.0-Linux-x86_64.sh -O miniconda.sh
 - bash miniconda.sh -b -p $HOME/miniconda
 - export PATH="$HOME/miniconda/bin:$PATH"
 - hash -r
 - conda config --set always_yes yes --set changeps1 no
 - conda update -q conda
 - conda info -a
 - |
-  conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION cytoolz numpy pandas pip pytables toolz psutil
+  conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION
 - source activate test-environment
-- pip install orca openmatrix zbox
-- pip install activitysim
 # - pip install https://github.com/ActivitySim/activitysim/zipball/dev/
 - pip install pytest pytest-cov coveralls pycodestyle
-- pip install sphinx numpydoc sphinx_rtd_theme
+# - pip install sphinx numpydoc sphinx_rtd_theme
```

Comment:
we currently don't build the docs with travis. should we?

Reply:
This should build the docs:
```
after_success:
- coveralls
- bin/build_docs.sh
```

Reply:
oh! that makes sense. `build_docs.sh` handles these installations, so I'll remove this line completely.